frontend/src/Map.js

```diff
@@ -62,15 +69,23 @@ class Map extends Component {
             });
         }
 
+        let delayFactor = 0;
+        let request = {
+            origin: start,
+            destination: finish,
+            waypoints: waypts,
+            optimizeWaypoints: true,
+            travelMode: window.google.maps.TravelMode.DRIVING
+        };
 
+        this.m_get_directions_route(delayFactor, request, directionsRenderer, directionsService);
+
+        this.props.onMapLoad(map)
+    }
+
+    m_get_directions_route(delayFactor, request, directionsRenderer, directionsService) {
```

Comment:
tbh, it's unclear for me from the name, what this function does. Consider renaming to something more clear

Reply:
It generates the routes from the api, sure, I will change the name